### DISCUSSION TITLE
Is pycaret valid in Python 3.11?

### DISCUSSION THREAD

**Discussion by harleiaki** [pycaret/pycaret#3778] [Q&A]
Is pycaret valid in Python 3.11? 
![image](https://github.com/pycaret/pycaret/assets/96266332/9e1b0450-ecc8-4855-94cf-7e45104c27d1)

**Comment by celestinoxp** [comment]
At this moment pycaret does not yet support python 3.11. The maximum supported version is Python 3.10

**Comment by celestinoxp** [comment]
@harleiaki now yes, pycaret already supports python 3.11. I'm even trying to increase the version of dependencies such as pandas 2.0 among others...
